src/core/androidplatformutilities.cpp

```diff
@@ -260,6 +260,7 @@ bool AndroidPlatformUtilities::checkAndAcquirePermissions( const QString &permis
 
 void AndroidPlatformUtilities::setScreenLockPermission( const bool allowLock )
 {
+  /*
```

Comment:
That, my friend, was not meant for this pull request!

Reply:
Argh... not the first time happening.